internal/capability/http/http.go

```diff
@@ -44,7 +45,9 @@ func (httpCapability *HttpCapability) Execute(
 	command cacao.Command,
 	authentication cacao.AuthenticationInformation,
 	target cacao.AgentTarget,
-	variables cacao.Variables) (cacao.Variables, error) {
+	variables cacao.Variables,
+	inputVariableKeys []string,
```

Comment:
Am I missing something, or are inputvariablekeys never used in the Execute function? This comment applies to all capabilities

Reply:
That is true as there is no use for it in the spec at this point. We will need to see how we can implement this for now I wanted to include it in the interface as changing it again is a lot of work.